go/integTest/standalone_commands_test.go

```diff
@@ -388,3 +388,10 @@ func (suite *GlideTestSuite) TestSortReadOnlyWithOptions_SuccessfulSortByWeightA
 	assert.Equal(suite.T(), "item1", sortResult[3].Value())
 	assert.Equal(suite.T(), "item3", sortResult[5].Value())
 }
+
+func (suite *GlideTestSuite) DBSize_Success() {
```

Comment:
This test doesn't run.
```suggestion
func (suite *GlideTestSuite) TestDBSize() {
```

Reply:
resolved 392d540efbc2114f6839af9dd96492495d464019